core-v1/src/main/scala/algebra/Expr.scala

```diff
@@ -1252,7 +1270,7 @@ object Expr {
   )(implicit
     zip: ZipToShortest.Aux[W, WL, OP, UL],
     opO: OP[W[UL]],
-  ) extends Expr[I, W[UL], OP]("zipToHList") {
+  ) extends Expr[I, W[UL], OP]("zipToShortest") {
```

Comment:
This was a misnomer from before. Now that there are 2 expressions that do similar things, I need to distinguish them.